### ISSUE TITLE
Server-timing not shown on Windows?

### ISSUE THREAD

**Issue by tweidema** [riok/Kreya#262]
**Describe the bug**
I have implemented a gRPC service returning a server-timing header as described here: https://kreya.app/blog/kreya-1.12-whats-new/#operation-and-server-timing-
On Kreya 1.15 on linux Kreya shows a Server Timing section in the Timing tab, with the fields set in the header (as shown in the link above).
But on Kreya 1.15 on Windows this is not shown. Both on Linux and on Windows you can see the server-timing header under the "Header" tab for the call.

**To Reproduce**
See above.

**Expected behavior**
The Server Timing info to be shown in Windows version also.

**Screenshots**

**Environment (if possible, copy the information from the error dialog or the About menu):**
 - OS: Windows 11 PRO 23H2
 - Kreya Version 1.15.0

**Comment by CommonGuy** [comment]
Can you test with the Kreya example project (this one: https://github.com/riok/Kreya/tree/master/example-project)?

I'm on Windows too and it works for me:
![grafik](https://github.com/user-attachments/assets/c9cc7e2b-a1e1-4a8b-bbfa-51fccee776a5)

**Comment by tweidema** [comment]
With the example-project the Server-Timings section is shown on windows as it should be. Strange.
The example-project sends this server-timing header:
```
db;dur=53, app;dur=417.2, cache;dur=123.2, total;dur=593.4
```
Our own project, where it is only shown on linux (even though as I wrote it is visible in the header-tab) sends this:
```
gRPC;dur=27.83489227294922
```
Look valid to me?

**Comment by CommonGuy** [comment]
Yes it looks valid. If I try that header, it also works for me... Could you show a screenshot? Maybe we missed something else... What type of gRPC operation is it?

**Comment by tweidema** [comment]
It's a simple unary gRPC call, no streaming. I will try to get the Windows-user make screenshots when she comes in, but here is what it looks like on linux (where it works fine):
![linux_header](https://github.com/user-attachments/assets/0d0777a8-18e9-4f6d-be2e-f25f781c2c9d)
![linux_timing](https://github.com/user-attachments/assets/dc88d670-9bba-4ee6-9f4f-b8dfad9fe062)

**Comment by tweidema** [comment]
And this is what it looks like on Windows 11 PRO 23H2
![T3](https://github.com/user-attachments/assets/c1f6888d-56f7-4ff0-9871-5ac9443743f3)
![T5](https://github.com/user-attachments/assets/ac8f63b7-e341-4c8b-bea2-427ac7190830)

**Comment by CommonGuy** [comment]
Thanks for the quick replies, I'll try to reproduce this

**Comment by CommonGuy** [comment]
I was finally able to reproduce and fix this. The problem was that we parsed the duration based on the user culture. In this case, the culture was set to something that has a period as the thousand separator, which caused the number to be parsed incorrectly. The resulting duration was too large, which is why it wasn't being displayed.

The user culture should obviously not be respected in this case. The bugfix will be included in the next release

**Comment by tweidema** [comment]
I have been bitten by that very thing as well. Great you found it.

**Comment by CommonGuy** [comment]
Included in Kreya 1.16, which was just released
